var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -192,15 +214,26 @@ def cmake_args(self):
             '-DTrilinos_ENABLE_EXPLICIT_INSTANTIATION:BOOL=ON',
             '-DTrilinos_ENABLE_CXX11:BOOL=ON',
             '-DTPL_ENABLE_Netcdf:BOOL=ON',
-            '-DTrilinos_ENABLE_Tpetra:BOOL=%s' % (
-                'ON' if '+tpetra' in spec else 'OFF')
+            '-DNetCDF_ROOT:PATH=%s' % spec['netcdf'].prefix,
```

Comment:
In my configure script I don't need to specify the path to packages, so maybe this is not needed.